API/health/controllers/health_controller.py

```diff
@@ -1,9 +1,24 @@
 from flask import Blueprint
 import flask
+#from flask_httpauth import HTTPBasicAuth
+from API.authentication.auth import auth
 
 health_blueprint = Blueprint('health', __name__)
+#auth = HTTPBasicAuth()
 
+# USER_DATA = {
```

Comment:
I would remove commented zone and everything that relates to authentication logic from this file